.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+#
+# require both blockchain-team-codeowners and blockchain-team to review all PR's not specifically matched below
+* @stacks-network/blockchain-team-codeowners @stacks-network/blockchain-team
+
+
+# Signer code
+# require both blockchain-team and blockchain-team-signer to review PR's for the signer folder(s)
+libsigner/**/*.rs @stacks-network/blockchain-team-codeowners @stacks-network/blockchain-team-signer
+stacks-signer/**/*.rs @stacks-network/blockchain-team-codeowners @stacks-network/blockchain-team-signer
```

Comment:
Comment says `blockchain-team` but line says `blockchain-team-codeowners`.

Reply:
addressed in 9b4d8995f73c86501f15b8525944ea9c59a102d6